docs/logging/logging.md

```diff
@@ -427,4 +447,230 @@ spec:
     url: http://loki-stack.cattle-logging-system.svc:3100
     extra_labels:
       logOutput: harvester-loki
-```
\ No newline at end of file
+```
+
+## Audit
+
+Harvester collects Kubernetes `audit` and is able to send the `audit` to vairous types of log server.
+
+The policy file to guide `kube-apiserver` is [here] (https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rke2-92-harvester-kube-audit-policy.yaml).
```

Comment:
```suggestion
The policy file to guide `kube-apiserver` is [here](https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rke2-92-harvester-kube-audit-policy.yaml).
```

Reply:
Suggest using permalink in case it's removed.

Reply:
Updated with permalink. thanks.